Add ux principles and content guidelines (#7)
Shopify/channels-ui-docs: Content guidelines.md

# Content guidelines

We know that your platform has its own voice and tone, and it’s ok to preserve them in the channel. However, channels are part of Shopify and merchants don’t see them as a separate experience.

When you’re creating content for a sales channel, use Shopify’s [voice and tone guide](https://polaris.shopify.com/content/voice-and-tone) to validate your decisions.

> > Note

Reviewer: ```
> Note:
> When building the channel, etc.
```

>

> > > When building the channel, use the names of features you have on your platform without altering, so that merchants can recognize them when they leave Shopify admin and use your platform. 

Reviewer: ```suggestion
> > When building the channel, don't change the names of features on your platform, so that merchants can recognize them when they leave the Shopify admin and use your platform. 
```

---

## Specific content use cases

Channels are a bridge between Shopify and your platform. To communicate correctly, you might require unique messaging and patterns that will differ from what merchants are used to seeing in Shopify. That’s okay.

---

## Communicating external errors

When merchants run into errors that are caused by something out of Shopify’s control, this is how you can communicate it.

> |                                                                                               |                                                                                                     |

Reviewer: Do we need Do/Don't  here? The alt text won't be visible on the image, it's just for screen readers I think

While merchants are setting up their channel they may need to take an action outside of Shopify. Make sure to be as clear as possible outlining merchant’s next steps.

|                                                                                                                         |                                                                                                                               |
| ----------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------- |
| ![Example of how to indicate that action is required ourside of Shopify](images/do-action-required-outside-shopify.png) | ![Example of how not to indicate that action is required outside of Shopify](images/dont-action-required-outside-shopify.png) |

**Best practices:**

1. Tell merchants where they need to go to resolve the issue, and be clear about when they are leaving Shopify.
2. Provide a direct link or guide them.
3. Tell merchants what they need to do after they leave Shopify.

---

## Important channel updates

> If there are new requirements for your channel or important information that merchants need to know, communicate them as early and as directly as possible. Mention the impact on the merchant’s store or sales, if any. Don’t rely on email or external communication, as merchants often miss it.

Reviewer: ```suggestion
If your channel has new requirements or there's other important information that merchants need to know, tell merchants as early and as directly as possible. Mention the impact on the merchant’s store or sales, if any. Don’t rely on email or external communication, as merchants often miss it.
```

|                                                                                           |                                                                                                 |
| ----------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------- |
| ![Example of how to communicate channel updates](images/do-important-channel-updates.png) | ![Example of how not to communicate channel updates](images/dont-important-channel-updates.png) |

**Best practices:**

1. Tell the merchants if any action is required of them.

> 2. When possible, add a relevant date or time.

Reviewer: ```suggestion
2. Where possible, add a relevant date or time.
```

Reviewer: Should we instantiate with an example? It's a little ambiguous as currently phrased, but it might be clear to devs

Author: updated the content here a bit for clarity

3. Let merchants know how long this announcement may affect them and if there are any consequences.

> 4. If there’s ever impact on their store or sales, be upfront.

Reviewer: ```suggestion
4. If there’s ever an impact on the merchants store or sales, be upfront about it.
```
